cdap-api-common/src/main/java/co/cask/cdap/api/data/schema/Schema.java

```diff
@@ -732,22 +729,27 @@ private boolean checkCompatible(Schema target, Multimap<String, String> recordCo
   }
 
   /**
-   * Creates a map of indexes based on the iteration order of the given set.
+   * Creates a pair of Maps such that the first map contains indexes based on the iteration order of the given set.
+   * The second map is the reverse of the first map.
```

Comment:
Should we more explicitly say what is the reverse of the map? something like - `The second map contains the same entries as the first map but with reversed keys and values.`